src/components/Checkout/LoanPrice.vue

```diff
@@ -114,9 +122,10 @@ export default {
 
 				const minAmount = parseFloat(this.minAmount || 25); // 25_hard_coded
 
-				const priceArray = getDropdownPriceArrayCheckout(remainingAmount, minAmount, this.enableFiveDollarsNotes); // eslint-disable-line max-len
-				if (!priceArray.includes(Number(this.price).toFixed())) {
-					priceArray.push(Number(this.price).toFixed());
+				const showHugeAmount = this.enableHugeAmount && this.isLoggedIn;
+				const priceArray = getDropdownPriceArrayCheckout(remainingAmount, minAmount, this.enableFiveDollarsNotes, showHugeAmount); // eslint-disable-line max-len
+				if (!priceArray.includes(numeral(this.price).format('0,0'))) {
```

Comment:
Here you changed the formatting. Why was that needed?

Reply:
Mainly for the case of $1,000 coming from shared loan cards. Not affecting other cases.

Reply:
Does it affect the existing formatting? And the comparison still works, since the formatting used to be different?

Reply:
I tested different situations and comparison still works for current and new cases.